src/main/java/Deadline.java

```diff
@@ -0,0 +1,15 @@
+public class Deadline extends Task{
```

Comment:
Able to see the different OOP features being implemented. Also, good to add a space before '{'.